test/functionalTest/cases/1048_subscription_cache/three_subscriptions.test

```diff
@@ -25,7 +25,7 @@ notifyFormat_ngsi10
 
 --SHELL-INIT--
 dbInit CB
-brokerStart CB 38,101 IPv4 -statSemWait
+brokerStart CB 38,101 IPv4 -statSemWait --cache
```

Comment:
I'd suggest to use `-cache` to align with the same style used for other parameters in `brokerStart` (e.g. `-statSemWait`).